scilifelab/report/survey.py

```diff
@@ -61,6 +61,17 @@ def project_closed(project, format="%Y-%m-%d"):
     except:
         pass
     return date
+
+def project_aborted(project):
```

Comment:
I'll try to remember that. Maybe I was thinking in ruby where you can do this
`def project_aborted?(project)`